<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmQuotesIBM</class>
 <widget class="QDialog" name="frmQuotesIBM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Añade una cotización manualmente</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblInversion">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 192, 0);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCalendarWidget" name="calendar">
       <property name="firstDayOfWeek">
        <enum>Qt::Monday</enum>
       </property>
       <property name="verticalHeaderFormat">
        <enum>QCalendarWidget::NoVerticalHeader</enum>
       </property>
       <property name="dateEditEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="timeLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Introduce la hora</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="txtTime">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="chkNone">
         <property name="text">
          <string>Cotización de cierre</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Introduce la cotización</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="myQLineEdit" name="txtQuote"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Introduce la zona horaria</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbZone"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="cmd">
       <property name="text">
        <string>Guardar cotización</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">myqlineedit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
